og.info.yml

```diff
@@ -6,6 +6,8 @@ core: 8.x
 type: module
 
 dependencies:
+  - entity_reference
```

Comment:
Are we dependent on this module? I thought all the stuff we are using is core itself. Not core entity reference now.